app/flatpak-builtins-create-usb.c

```diff
@@ -639,7 +641,7 @@ flatpak_builtin_create_usb (int argc, char **argv, GCancellable *cancellable, GE
         if (deploy_data == NULL)
           return FALSE;
 
-        if (flatpak_deploy_data_has_subpaths (deploy_data))
+        if (flatpak_deploy_data_has_subpaths (deploy_data) && !opt_allow_partial)
```

Comment:
Would be nice to mention `--allow-partial` in the error message. But also doesn't this mean most times `create-usb` is called it will fail, since only a subset of locales are installed by default and `create-usb` makes no effort to install the whole locale extension?

Reply:
We could mention it, but It seems non-ideal if people create a lot of usb repos that are broken for a random subset of people. Do we really expect a lot of people without much knowledge about the details to create sideload repos? I was thinking this warning is mostly a early check to avoid mistakes when e.g. endless want to create an install media for offline updates.